.vscode/launch.json

```diff
@@ -1,21 +1,21 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Cargo Run",
-            "env": {
-                "GOOGLE_API_KEY": "${env:GOOGLE_API_KEY}"
-            },
-            "type": "lldb",
-            "request": "launch",
-            "program": "${workspaceFolder}/target/debug/askrepo",
-            "args": ["./src", "-p", "'find bugs'"],
-            "cwd": "${workspaceFolder}",
-            "preLaunchTask": "rust: cargo build",
-            "sourceLanguages": ["rust"]
-        }
-    ]
-}
\ No newline at end of file
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Cargo Run",
+      "env": {
+        "GOOGLE_API_KEY": "${env:GOOGLE_API_KEY}"
+      },
+      "type": "lldb",
+      "request": "launch",
+      "program": "${workspaceFolder}/target/debug/askrepo",
+      "args": ["./src", "-p", "'find bugs'"],
+      "cwd": "${workspaceFolder}",
+      "preLaunchTask": "rust: cargo build",
+      "sourceLanguages": ["rust"]
+    }
+  ]
+}
```

Comment:
The Cargo-related files were created by an old build environment. Let's delete them.